### ISSUE TITLE
Config file not created and adjustments

### ISSUE THREAD

**Issue by RubyCascade** [JustDoom/ItsStillExplosive#3]
Bug:
The file for the config mentioned in [Config option for affected items](https://github.com/JustDoom/DoomsItsStillExplosive/issues/1) is not created upon loading the game with the mod installed.

Adjustment suggestions:
- Add stack multiplier to the config _(the amount the base explosion power is multiplied by for each additional item in the stack)_
- increase default stack multiplier _(probably to around 75-80% instead of the around 20% it's at now (or just make it 100% by default. lol idk))_
- increase default TNT item explosion strength to at least half of placed TNT _(a stack of TNT is_ **way** _too weak, roughly equivalent to 1 placed)_
- add End Crystals to the list of items that explode by default _(yes, I know fire doesn't set off placed end crystals but a punch does and end crystal items burn)_

**Comment by JustDoom** [comment]
What modloader, Minecraft and mod version are you using?

**Comment by RubyCascade** [comment]
Minecraft version: 1.20.1
Modloader: Fabric 0.16.9
Mod version: v1.1.0

**Comment by JustDoom** [comment]
Are you sure you are in the right location? And that the mod is actually loaded without errors. It generates for me
![image](https://github.com/user-attachments/assets/8dd7b605-99a9-484f-a3cc-abeda53858ed)

**Comment by JustDoom** [comment]
@RubyCascade I will try add the new config options as well soon

**Comment by RubyCascade** [comment]
Ah, it does generate. For whatever reason I wasn't able to find it even when I sorted by date created. Sorry about that lol

**Comment by RubyCascade** [comment]
also it would be nice if you switched to something like json or toml, .properties looks like a (small) nightmare to edit. _YetAnotherConfigLib (YACL)_ and _Forge Config API Port_ are popular config libraries that you might want to take a look at.

**Comment by JustDoom** [comment]
Ah glad you sorted it out! I will look into a better config system at some point. I need one that is modloader independent
 since I support a few of those at once.

**Comment by RubyCascade** [comment]
YACL is modloader independant, supporting Fabric, Forge, NeoForge, and Quilt. For 1.21 it only supports Fabric and NeoForge, but for 1.20 it supports all four modloaders except Quilt which is only (officially) supported in older versions of the mod. For 1.19 it has support for Fabric, Forge, and Quilt. https://modrinth.com/mod/yacl

Forge Config API Port is also _sort of_ modloader independent. It has more supported game versions and also supports the same modloaders as YACL except Quilt is dropped after 1.19 (same as YACL) and Forge/NeoForge isn't supported until 1.20.2 (the first version of NeoForge is 1.20.1) due to it's design. https://modrinth.com/mod/forge-config-api-port
> Forge Config API Port is a modding library for mod developers that provides both NeoForge's & Forge's whole config system to other modding ecosystems. Allows developers to use NeoForge configs on Fabric & Forge and Forge configs on Fabric & NeoForge.
> 
> The library is built with a multiloader architecture in mind by also offering a mod loader independent common distribution, so there needs to be as little platform specific code as possible.

YACL is the more popular and customizable of the two.

**Comment by JustDoom** [comment]
Yeah that is another issue, a lot are not updated too the latest version which is annoying
